core/src/main/java/com/dooapp/fxform/controller/PropertyEditorController.java

```diff
@@ -90,7 +93,8 @@ public void changed(ObservableValue observableValue, Object o, Object o1) {
                 }
             }
         };
-        fxFormNode.getProperty().addListener(viewChangeListener);
+        weakViewChangeListener = new WeakChangeListener(viewChangeListener);
+        fxFormNode.getProperty().addListener(new WeakChangeListener(weakViewChangeListener));

```

Comment:
You added a WeakChangeListener of a WeakChangeListener here. I suppose you wanted to do instead `fxFormNode.getProperty().addListener(weakViewChangeListener);`
Also for `weakModelChangeListener `

Reply:
You are right, this has been fixed.